ctrd/image_proxy_util.go

```diff
@@ -123,6 +123,9 @@ func useProxy(addr string) bool {
 		addr = addr[:strings.LastIndex(addr, ":")]
 	}
 
+	// There is no standard way of pattern matching for environmental variable no_proxy.
+	// The implementation below is similar to curl's.
+	// Here is a discussion of the syntax of no_proxy: https://github.com/curl/curl/issues/1208
```

Comment:
Maybe this is more appropriate in your own notes. Anyway, thank you for your contribution.